bears/general/aspectsYEAHBear.py

```diff
@@ -0,0 +1,67 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bearlib.aspects.Spelling import aspectsYEAH
+from coalib.bearlib.languages import Unknown
+
+
+class aspectsYEAHBear(LocalBear, aspects={
+        'detect': [aspectsYEAH],
+        'fix': [aspectsYEAH], }):
+    LANGUAGES = {'All'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Spelling'}
+
+    def run(self, filename, file, aspects=[
+            aspectsYEAH(Unknown),
+    ]):
+        """
+        Check for the correct spelling of ``aspect`` and ``aspectsYEAH``
+        in the file.
+        """
+
+        for aspect in aspects:
+            if type(aspect) is aspectsYEAH:
```

Comment:
We only need to implement a more generalized way of running bear logic for supported aspects than in such a `for` loop with several `if type(aspect) is ...` checks